app/models/project_services/teamcity_service.rb

```diff
@@ -115,13 +115,13 @@ def commit_status(sha)
     end
   end
 
-  def execute(data)
+  def execute(push)
     auth = {
       username: username,
       password: password,
     }
 
-    branch = data[:ref]
+    branch = push[:ref].gsub("refs/heads/", "")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.